src/VerifyAccountHandler.ts

```diff
@@ -35,14 +36,23 @@ export class VerifyAccountHandler implements LambdaInterface {
 		try {
 			const { sessionId, body, encodedHeader } = this.validateEvent(event);
 			const clientIpAddress = event.headers[Constants.X_FORWARDED_FOR] ?? event.requestContext.identity?.sourceIp;
+			const credentialVendorSsmPath = checkEnvironmentVariable(EnvironmentVariables.CREDENTIAL_VENDOR_SSM_PATH, logger);
+			CREDENTIAL_VENDOR = await getParameter(credentialVendorSsmPath);
+			if (CREDENTIAL_VENDOR === "HMRC") {
+				const hmrcTokenSsmPath = checkEnvironmentVariable(EnvironmentVariables.HMRC_TOKEN_SSM_PATH, logger);
+    			VENDOR_TOKEN = await getParameter(hmrcTokenSsmPath);
+
+				logger.appendKeys({ sessionId });
+				logger.info("Starting VerifyAccountRequestProcessorHmrc");
+				return await VerifyAccountRequestProcessor.getInstance(logger, metrics, VENDOR_TOKEN).processHmrcRequest(sessionId, body, clientIpAddress, encodedHeader);
+			} else {
+				const experianTokenSsmPath = checkEnvironmentVariable(EnvironmentVariables.EXPERIAN_TOKEN_SSM_PATH, logger);
+				VENDOR_TOKEN = await getParameter(experianTokenSsmPath);
+				logger.appendKeys({ sessionId });
+				logger.info("Starting VerifyAccountRequestProcessorExperian");
+				return await VerifyAccountRequestProcessor.getInstance(logger, metrics, VENDOR_TOKEN).processExperianRequest(sessionId, body, clientIpAddress, encodedHeader);
```

Comment:
I think we need an additional check on the get instance method. For instance if I have created an HMRC instance and then I call this. Because there is an instance existing I believe it would return the HMRC rather than the Experian instance